app/src/main/java/org/oppia/android/app/databinding/ConstraintLayoutAdapters.java

```diff
@@ -7,10 +7,14 @@
 import androidx.constraintlayout.widget.ConstraintSet;
 import androidx.databinding.BindingAdapter;
 
-/** Custom binding adapters that set constraints to views. */
-public final class ConstraintViewAdapters {
+/**
+ * Custom binding adapters that set constraints to views.
+ */
```

Comment:
Compress this to a single line, e.g.: ``/** ... */`` per https://developer.android.com/kotlin/style-guide#formatting_2.

Ditto elsewhere in the PR.

Reply:
Done